charts/nextcloud/1.6.17/templates/deployment.yaml

```diff
@@ -82,6 +82,20 @@ spec: {{ include "common.deployment.common_spec" . | nindent 2 }}
         - name: extrappvolume-{{ $index }}
           mountPath: {{ $hostPathConfiguration.mountPath }}
         {{ end }}
+        {{- if .Values.nextcloud.install_ffmpeg }}
+        lifecycle:
+          postStart:
```

Comment:
Should we have a probe perhaps to make sure not to mark the app as ready when this is being installed?

Reply:
This runs after probe greenlights it.
Helm does not support adding your own custom states between deploying and ready.

There is a reason this is a bad idea.

Reply:
@stavros-k can we see if we can wait for this in a probe..
https://kubernetes.io/docs/concepts/containers/container-lifecycle-hooks/#container-hooks it appears it should be independent of probes 🤔

Reply:
> Should we have a probe perhaps to make sure not to mark the app as ready when this is being installed?

I've added probes, I only check on the startupProbe for the ffmepg command.
On liveness and readiness I do a check on the status.php page